app/utils/url.js

```diff
@@ -1,16 +1,18 @@
-import DeviceInfo from 'react-native-device-info';
+/* eslint-disable max-classes-per-file */
+import { getLocales } from 'react-native-localize';
+import { Platform } from 'react-native';
 
 let server = 'https://thalia.nu';
 /* istanbul ignore next line */
 if (__DEV__) { // eslint-disable-line no-undef
-  server = 'http://localhost:8000';
+  server = Platform.OS === 'android' ? 'http://10.0.2.2:8000' : 'http://localhost:8000';
```

Comment:
Why that IP address?

Reply:
On Android localhost will now point to the device itself instead of to the computer you are running it on. The computer will always have 10.0.0.2 as IP address so we have to replace localhost by 10.0.0.2 in development but only for the android side of things.

Reply:
Huh? Always 10.0.0.2? That is new to me. The idea of localhost is that the adb will route the 8000 port via a reverse proxy to the device. I suspect that the gradle changes removed that line on the before run though.